tests/template_literals/styled-jsx-with-expressions.js

```diff
@@ -23,7 +221,7 @@
     color: red;
   }
   }
-  @font-face {
+@font-face {
```

Comment:
also this is not right, seems this at-rule with 
`a:placeholder;` (we are faking single placeholder to this) cant parse as expected, is there other thing we can fake? tried  fake into at-rule and comment , some other problem come up

Reply:
misstaken, not snap